MVVMRxSwiftDemo/Sources/Supports/Extensions/UIKit/UIImage+Background.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  UIImage+Background.swift
+//  MVVMRxSwiftDemo
+//
+//  Created by Chananchida F. on 7/27/21.
+//
+
+import UIKit
+
+extension UIImage {
+
+    static func withBackground(color: UIColor, size: CGSize = CGSize(width: 1, height: 1)) -> UIImage? {
+        let renderer = UIGraphicsImageRenderer(size: size)
+        
+        return renderer.image { ctx in
+            color.setFill()
+            ctx.fill(CGRect(x: 0, y: 0, width: size.width, height: size.height))
```

Comment:
float should have decimal 
```suggestion
            ctx.fill(CGRect(x: 0.0, y: 0.0, width: size.width, height: size.height))
```